db/data_migrations/20241024131255_migrate_data_request_areas.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+class MigrateDataRequestAreas < ActiveRecord::DataMigration
+  def data_request_area_type_for(request_type)
+    case request_type
+    when *DataRequest::BRANSTON_DATA_REQUEST_TYPES
+      "branston"
+    when *DataRequest::BRANSTON_REGISTRY_DATA_REQUEST_TYPES
+      "branston_registry"
+    when *DataRequest::MAPPA_DATA_REQUEST_TYPES
+      "mappa"
+    when *DataRequest::PRISON_DATA_REQUEST_TYPES + %w[cctv_and_bwcf]
+      "prison"
+    when *DataRequest::PROBATION_DATA_REQUEST_TYPES + %w[court]
+      "probation"
+    when "other"
+      "other_department"
+    end
+  end
+
+  def up
+    # get DataRequests for only Offender and OffenderComplaint
+    data_requests = DataRequest.joins(:offender_sar_case)
```

Comment:
I checked on production, and there are no data requests that aren't associated to these case types, so this step isn't necessary.